src/pretalx/settings.py

```diff
@@ -11,28 +10,98 @@
 
 
 config = configparser.RawConfigParser()
+config.read_dict({
+    'filesystem': {
+        'base': os.path.dirname(os.path.dirname(__file__)),
+    },  # defaults depend on the data dir and need to be set once the data dir is fixed
+    'site': {
+        'debug': 'runserver' in sys.argv,
+        'url': 'http://localhost',
```

Comment:
does this somehow take the port into account later on?

Reply:
Nope, but I'd like to tackle that problem later if at all